include/net/golioth/settings.h

```diff
@@ -7,11 +7,15 @@
 #ifndef GOLIOTH_INCLUDE_NET_GOLIOTH_SETTINGS_H_
 #define GOLIOTH_INCLUDE_NET_GOLIOTH_SETTINGS_H_
 
-#include <qcbor/qcbor.h>
-#include <qcbor/qcbor_spiffy_decode.h>
 #include <net/coap.h>
 #include <stdint.h>
 
+struct _QCBOREncodeContext;
+struct _QCBOREncodeContext;
+
+typedef struct _QCBORDecodeContext QCBORDecodeContext;
+typedef struct _QCBOREncodeContext QCBOREncodeContext;
```

Comment:
Are these needed? Nothing in this file reference `QCBORDecodeContext` or `QCBOREncodeContext`.